### PR TITLE
926539 - Included the New Features of Maui Tools controls for Volume 4 2024 in UG.

### PR DESCRIPTION
**Pull request by Hemalatha-SF4675**
Hi team, 

I have included the New Features of Maui Tools controls for Volume 4 2024 in UG.

Regards,
Hemalatha

### CODE REVIEW

File: MAUI/Autocomplete/Selection.md [hunk 1]
```diff
@@ -394,4 +394,41 @@ In the Autocomplete control, the drop-down can be opened or closed programmatica
 autocomplete.IsDropDownOpen = true;
 
 {% endhighlight %}
+{% endtabs %}
+
+
+## Clear() method support
+
+Clear the selected values in the Autocomplete using the Clear() method as shown in the code snippet below:
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Selection.md:402]: Revamp the content.

File: MAUI/Autocomplete/Selection.md [hunk 2]
```diff
@@ -394,4 +394,41 @@ In the Autocomplete control, the drop-down can be opened or closed programmatica
 autocomplete.IsDropDownOpen = true;
 
 {% endhighlight %}
+{% endtabs %}
+
+
+## Clear() method support
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Selection.md:400]: Remove ()

File: MAUI/Autocomplete/Selection.md [hunk 3]
```diff
@@ -394,4 +394,41 @@ In the Autocomplete control, the drop-down can be opened or closed programmatica
...
+Clear the selected values in the Autocomplete using the Clear() method as shown in the code snippet below:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autocomplete"
+            WidthRequest="250"
+            ItemsSource="{Binding SocialMedias}"
+            IsDropDownOpen = true;
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Selection.md:411]: IsDropDownOpen not needed here.

File: MAUI/Autocomplete/Selection.md [hunk 4]
```diff
@@ -394,4 +394,41 @@ In the Autocomplete control, the drop-down can be opened or closed programmatica
...
 
 {% endhighlight %}
+{% endtabs %}
+
+
+## Clear() method support
+
+Clear the selected values in the Autocomplete using the Clear() method as shown in the code snippet below:
+
+{% tabs %}
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Selection.md:404]: Just add the code used to clear items. No need of elongated codes.

File: MAUI/ComboBox/Selection.md [hunk 5]
```diff
@@ -356,4 +356,40 @@ In the .NET MAUI ComboBox control, the .NET MAUI drop-down list can be opened or
 comboBox.IsDropDownOpen = true;
 
 {% endhighlight %}
+{% endtabs %}
+
+## Clear() method support
```
Comment (PrithisIyyappan) [inline MAUI/ComboBox/Selection.md:361]: Same concerns in autocomplete.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 6]
```diff
@@ -232,6 +232,32 @@ The `NumericEntry` control supports customization of the UpDownButton's appearan
 
 ![UpDownButtonTemplate support in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonTemplate.png)
 
+## Up Down Button Alignment
+
+The Up Down Button Alignment feature specifies the alignment of the Up Down button in the NumericEntry controls.
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/UpDown-Button.md:237]: Revampl this content.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 7]
```diff
@@ -232,6 +232,32 @@ The `NumericEntry` control supports customization of the UpDownButton's appearan
...
+## Up Down Button Alignment
+
+The Up Down Button Alignment feature specifies the alignment of the Up Down button in the NumericEntry controls.  
+•   Left – Both the Spin buttons are positioned at the Left
+•   Right – Both the Spin buttons are positioned at the Right
+•   Both – Spin buttons are positioned at the extreme ends.
+
+N> The default value of this property is `Right`
+{% tabs %}
+{% highlight XAML %}
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/UpDown-Button.md:244]: Add snippets and output for left and both.

File: MAUI/Autocomplete/Selection.md (hunk 4 above)
Comment (Hemalatha-SF4675) [reply MAUI/Autocomplete/Selection.md:404]: Updated

File: MAUI/NumericEntry/UpDown-Button.md [hunk 8]
```diff
@@ -91,6 +91,48 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.InlineVerti
 
 ![UpDown Vertical Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacementVertical.png)
 
+## UpDown button alignment
+
+You can customize the `NumericEntry` control button alignment by using the `UpDownButtonAlignment` property. Set it to **Left** to position both buttons on the left, or **Both** to place them at the extreme ends.
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/UpDown-Button.md:96]: You can adjust the alignment of the UpDown buttons in the NumericEntry control using the UpDownButtonAlignment property. Set its value to Left, Right, or Both to position the buttons on the left, right, or both sides of the entry field, respectively.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 9]
```diff
@@ -91,6 +91,48 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.InlineVerti
...
+## UpDown button alignment
+
+You can customize the `NumericEntry` control button alignment by using the `UpDownButtonAlignment` property. Set it to **Left** to position both buttons on the left, or **Both** to place them at the extreme ends.  
+
+N> By default, the `UpDownButtonAlignment` property is set to **Right**.  
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfNumericEntry UpDownPlacementMode="Inline"
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/UpDown-Button.md:103]: Add TextAlignment changes in the Code.

File: MAUI/Autocomplete/Selection.md (hunk 1 above)
Comment (Hemalatha-SF4675) [reply MAUI/Autocomplete/Selection.md]: Changes are updated

File: MAUI/Autocomplete/Selection.md (hunk 2 above)
Comment (Hemalatha-SF4675) [reply MAUI/Autocomplete/Selection.md]: Changes are updated

File: MAUI/ComboBox/Selection.md (hunk 5 above)
Comment (Hemalatha-SF4675) [reply MAUI/ComboBox/Selection.md]: Changes are updated

File: MAUI/NumericEntry/UpDown-Button.md (hunk 6 above)
Comment (Hemalatha-SF4675) [reply MAUI/NumericEntry/UpDown-Button.md]: Changes are updated

File: MAUI/NumericEntry/UpDown-Button.md (hunk 7 above)
Comment (Hemalatha-SF4675) [reply MAUI/NumericEntry/UpDown-Button.md]: Changes are updated

File: MAUI/NumericEntry/UpDown-Button.md (hunk 8 above)
Comment (Hemalatha-SF4675) [reply MAUI/NumericEntry/UpDown-Button.md]: Updated

File: MAUI/NumericEntry/UpDown-Button.md (hunk 9 above)
Comment (Hemalatha-SF4675) [reply MAUI/NumericEntry/UpDown-Button.md]: Changes are updated

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**